### ISSUE TITLE
Public API documentation

### ISSUE THREAD

**Issue by Millu** [invoke-ai/InvokeAI#3985]

**Comment by psychedelicious** [comment]
We've had a lot of feedback from users who want to build on top of Invoke's server API. While this is exciting, it's not why we built the API. The API exists to serve the needs of the application, not developers.

We have autogenerated documentation for via OpenAPI spec (accessible at http://localhost:9009/docs), but this is as much as we plan to do for the HTTP API.

The big hurdle for building on top of the app is the socket.io implementation. We have no plans to document this except with comments in code. That is to say, the only docs for this are there to support further development of the invoke app - not to support external apps built on top of the invoke API.

In short, Invoke is an application, not a library or service.

With this context, we won't be investing further into API documentation.
